packages/expo-modules-core/src/ts-declarations/EventEmitter.ts

```diff
@@ -23,6 +23,15 @@ export type EventSubscription = {
  * Its implementation is written in C++ and common for all the platforms.
  */
 export declare class EventEmitter<TEventsMap extends EventsMap = Record<never, never>> {
+  /**
+   * A dummy private property with the given generic type. It is required for TypeScript to correctly infer this subtype.
+   * E.g. `useEvent` would not be able to infer the events map which results in accepting any string as the event name.
+   * @hidden
+   * @private
```

Comment:
We can use only one of the params, we treat them as an aliases for docs rendering:
* https://github.com/expo/expo/blob/cae3c598a6bbf444f6b0eaad2acf152d0b9f4369/guides/Expo%20Documentation%20Writing%20Style%20Guide.md?plain=1#L330-L331